pyaedt/emit_core/results/revision.py

```diff
@@ -372,34 +372,34 @@ def notes(self, notes):
         self.emit_project._emit_api.save_project()
 
     @property
-    def max_n_to_1_instances(self):
+    def n_to_1_limit(self):
         """
-        The maximum number of instances per band combination allowed to run for N to 1.
+        Maximum number of inteference combinations to run per receiver for N to 1.
         A value of 0 disables N to 1 entirely.
```

Comment:
```suggestion
        Maximum number of inteference combinations to run per receiver for N to 1.
        
        - A value of ``0`` disables N to 1 entirely.
```
I think having each of these as a bullet entry would help with comprehension.